tdm-fe/src/app/components/layout/WebServicesHeader.tsx

```diff
@@ -2,16 +2,19 @@ import '~/app/components/layout/scss/WebServicesHeader.scss';
 import Container from '@mui/material/Container';
 
 const WebServicesHeader = () => {
-    // Todo add the correcte text when available
     return (
         <div id="ws-header">
             <Container>
+                <p className="text" id="ws-header-pre-title">
+                    Chargez bos corpus et découvrez les résultats des services TDM
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
                    Chargez vos corpus et découvrez les résultats des services TDM
```